app/controllers/application_controller.rb

```diff
@@ -10,6 +11,18 @@ class ApplicationController < ActionController::Base
 
   add_flash_types :alert_with_description, :alert_with_items, :confirmation, :tried_to_publish, :tried_to_preview
 
+  rescue_from EditionAssertions::StateError do |e|
+    Rails.logger.warn(e.message)
```

Comment:
One thing we should do as part of this is just double check that this makes it to kibana. I get a bit confused whether it's supposed to or not. If it doesn't we might want to add a splash of request information to explain where it came from.

Reply:
Checked (see main comments on PR).